server/implementation-servlet/src/main/java/io/smallrye/graphql/entry/http/ExecutionServlet.java

```diff
@@ -136,6 +142,15 @@ private static JsonObject toJsonObject(String jsonString) {
         }
     }
 
+    private Map<String, Object> createContext(HttpServletRequest request) {
+        Map<String, Object> context = new HashMap<>();
```

Comment:
- Could you change this to add an entry with key `httpHeaders` and value of type `Map<String, List<String>>` (containing all http headers)? This is how we currently do it in the Quarkus implementation of this, so it would be a good idea to keep it unified: see https://github.com/quarkusio/quarkus/blob/3.7.0.CR1/extensions/smallrye-graphql/runtime/src/main/java/io/quarkus/smallrye/graphql/runtime/SmallRyeGraphQLAbstractHandler.java#L117
- I think the map needs to be concurrent because it then gets passed to operations that might run in parallel and write into it